ci/live-pipeline.yml

```diff
@@ -152,6 +152,8 @@ jobs:
               BUILD_TYPE: 0
               GH_TOKEN: ((github_access_token))
               SIGNING_KEY: ((github_ons_spp_machine_user_gpg_key_private))
+              SPACE_ID: ((contentful_space_id))
```

Comment:
These will need to be added to the build-files steps of each build type (dev branch, dev, preprod and prod) you may want to instead define these variables in env_setups: general at the top of the file, then here and the other build steps add <<: *general_params, just means each environment inherits the ID's from a single area (if we need separate ID's per environment there is specific environment settings at the top of the file as well)

Reply:
additionally can these changes be made to dev-pipeline.yml you can copy and paste the entire contents if you like to for ease just make sure that in the resources section, for each resource we have "check_every:" is set to never